rasa/core/channels/slack.py

```diff
@@ -268,21 +268,23 @@ def _sanitize_user_message(
         uids_to_remove = uids_to_remove or []
 
         for uid_to_remove in uids_to_remove:
+            escaped_uid = re.escape(uid_to_remove)
```

Comment:
We should add unit test which will check if re.escape is called.

Reply:
I've added a test to check for this